src/SvgParser.elm

```diff
@@ -285,18 +312,23 @@ parseToNode input =
 parse : String -> Result String (Html msg)
 parse input =
     let
-        toHtml svgNode =
-            case svgNode of
-                SvgElement element ->
-                    if element.name == "svg" then
-                        Ok <|
-                            svg (List.map toAttribute element.attributes)
-                                (List.map nodeToSvg element.children)
-
-                    else
-                        Err "Top element is not svg"
-
-                _ ->
+        toHtml svgNodes =
+            case svgNodes of
+                svgNode :: tl ->
+                    case svgNode of
+                        SvgElement element ->
+                            if element.name == "svg" then
+                                Ok <|
```

Comment:
You only use the first `SvgElement` from `svgNodes`. What is with the rest of it? If you want to remove comments please do it explicitly with it's own case and trigger an error if two `SvgElements` or a single `SvgText` are found in `svgNodes`.

Reply:
I've added a small comment to the docs to explain that we seek the first `svg` element. Let me know what you think

Reply:
I am okay with updating the docs. See my comments below.